dev-utils/src/commands/init_command.rs

```diff
@@ -0,0 +1,225 @@
+use super::genesis_command::GenesisCommand;
+
+use failure::Error;
+
+use super::genesis_dev_config::GenesisDevConfig;
+use read_input::prelude::*;
+use serde_json::json;
+use serde_json::Value as JsonValue;
+use std::fs;
+use std::fs::File;
+use std::io::prelude::*;
+use std::path::Path;
+use std::process::Command;
+use structopt::StructOpt;
+
+#[derive(Debug, StructOpt)]
+pub struct InitCommand {
+    data: String,
+    genesis_time: JsonValue,
+    app_hash: JsonValue,
+    app_state: JsonValue,
+    genesis_dev: JsonValue,
```

Comment:
I guess this one could use the struct directly: https://github.com/crypto-com/chain/pull/238/files#r308541001

i.e. `genesis_dev: GenesisDevConfig` instead of `JsonValue`

Reply:
ok, i will fix it

Reply:
removed JsonValue